{{cookiecutter.github_repository}}/provisioner/roles/nginx/templates/site.443.conf.j2

```diff
@@ -2,6 +2,7 @@
 
 server {
     listen       {{ ngnix_listen }};
+    listen       [::]:{{ ngnix_listen }};
```

Comment:
out of curiosity, `[::]` is a specific operator/identifier?

Reply:
It's a identifier.